packages/amplify-category-auth/src/provider-utils/supported-services.ts

```diff
@@ -1204,7 +1204,7 @@ const supportedServices = {
         key: 'signinwithappleClientIdUserPool',
         prefix:
           " \n You've opted to allow users to authenticate via Sign in with Apple. If you haven't already, you'll need to go to https://developer.apple.com/account/#/welcome and configure Sign in with Apple. \n",
-        question: 'Enter your Services ID for your OAuth flow: ',
+        question: 'Enter your Apple Services ID for your OAuth flow: ',
```

Comment:
You'll need to update all E2E tests that rely on this string, 
![image](https://user-images.githubusercontent.com/110861985/198719027-d6cea3f2-2c9e-4008-a514-0828e7621c0b.png)